physionet-django/project/migrations/0030_auto_20190807_1220.py

```diff
@@ -3,18 +3,19 @@
 from django.db import migrations, models
 from project.models import PublishedProject
 
-def featured(apps, schema_editor):
-    projects = PublishedProject.objects.filter()
+
+featured = {}
```

Comment:
What is the use of this "featured" ?

Reply:
This is a dict that will save the projects currently flagged as featured and assign an integer which will be used to populate the field again after it is recreated.